tokio/src/sync/rwlock.rs

```diff
@@ -241,6 +241,11 @@ impl<T> RwLock<T> {
 
         RwLockWriteGuard { lock: self, permit }
     }
+
+    /// Consumes the lock, returning the underlying data.
+    pub fn into_inner(self) -> T {
```

Comment:
Do you think there should also be a `get_mut` method [à la `std::sync::RwLock`'s](https://doc.rust-lang.org/std/sync/struct.RwLock.html#method.get_mut)? Not a blocker for this PR.

Reply:
Thanks for the review. I'd say it won't hurt. I suspect that sooner or later somebody will want it and having it will make it more likely that different `RwLock` implementations (with different trade-offs) can be swapped out more easily (because most mirror `std::sync::RwLock`'s API).

That being said, I don't need it right now and would like the pull request to not tackle multiple things. Given that the request has been approved, what needs to happen to merge it?

Reply:
Now that it's had two approvals, I can just go ahead and merge. The `get_mut` change definitely doesn't need to be merged in this branch, I just thought it was worth mentioning.